src/main/java/seedu/address/ui/UserProfileWindow.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.person.User;
+
+public class UserProfileWindow extends UiPart<Stage> {
+
+    private static final Logger logger = LogsCenter.getLogger(UserProfileWindow.class);
+    private static final String FXML = "UserProfileWindow.fxml";
+
+    private static final String DUMMY_PROFILE = "Komyo San\n" +
+            "phone: 81210395\n" +
+            "email: komyosan@gmail.com\n" +
+            "address: 100 Sin Ming Avenue\n" +
+            "telegramHandle: @komyosan\n" +
+            "groups: [Study, BrightHill]\n" +
+            "modules: [BT3101, CS3282, CS2107, CS3247, CE3115, IS3240]";
```

Comment:
By right, the default user should be LINUS, since the central focus is NUS students. But if this doesn't interfere with any test cases then it should be all fine. LGTM.

Reply:
Also be careful about the User implementation, cos Russell made it a singleton object, so whenever a new User is created, the rest are supposed to be destroyed.

Reply:
Hi, I would like to clarify, a Singleton object just means that there is one instance of User in the entire application. A new User is only ever created _once_ and hence there is no "rest" which can be destroyed.

Reply:
Dummy profile has been removed, it was there for testing, the user will be used instead.

Reply:
Yes, but the User's information is immutable, so the only way to change it is by destroying the user and creating a new one.